xapian-applications/omega/docs/postprocessing.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Log Data Postprocessing
+=========================
+
+Log data includes a) data logged from the search results page rendered by
+Omega query template and, b) clicks recorded when a document link is
+clicked on the results page.
+
+Further processing of this raw data is handled by the ``postprocess.py``
+script. This script generates the final clickstream log file from the input
+search and click log files which can be used to train click models for clickstream
+data mining. It also creates a Query file that can be used by Xapian Letor module
+for generating its training files.
+
+The two functions defined in ``postprocess.py`` are:
```

Comment:
This just makes me itchy to turn all this documentation into a proper sphinx project, so we can use its python documentation support. Let's hold off for now, but can you note that we want to do that? Probably when writing up the next step, which generates data for training letor, would be a good time.